docker/hooks/build

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+cd ../
+docker build -t olbase:latest --build-arg DOCKER_HUB=TRUE -f docker/Dockerfile.olbase .
```

According to the docs, this will let us control the image name/tag from docker hub ( https://docs.docker.com/docker-hub/builds/advanced/#environment-variables-for-building-and-testing )

```suggestion
docker build -t $IMAGE_NAME --build-arg DOCKER_HUB=TRUE -f docker/Dockerfile.olbase .
```

---

@cdrini I'm pushing to `martiansoul/openlibrary` and I edited `Dockerfile.oldev` and the site runs. :) you can also try pulling the image `martiansoul/openlibrary:latest` and editing `Dockerfile.oldev` yourself.